CondCore/PCLConfigPlugins/test/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Save current working dir so img can be outputted there later
+W_DIR=$(pwd);
+# Set SCRAM architecture var
+SCRAM_ARCH=slc6_amd64_gcc630;
+export SCRAM_ARCH;
+source /afs/cern.ch/cms/cmsset_default.sh;
+eval `scram run -sh`;
+# Go back to original working directory
+cd $W_DIR;
+# Run get payload data script
+
+####################
+# Test Gains
+####################
+/afs/cern.ch/user/c/condbpro/public/BROWSER_PI/getPayloadData.py \
```

Comment:
hi @ggovi - can you make that PR for getPayloadData now?

Reply:
what are the use cases for this thing? The password of the direct oracle access won't be available to the users anyways...

Reply:
@ggovi, as of now, mostly for those who have direct oracle (i.e. developers) to test the code when making changing and also to use the tool in local check-outs, which is still useful to produce plots.
Anyway, what prevents to provide direct oracle (reading) access to to the cms-bot to make this become part of a unit test? Incidentally I think we could use the same mechanism to test the O2Os.

Reply:
Provide read-access to the cms-bot means essentially store the password in the repository... Not viable.

Reply:
The O2O tests could be done with frontier+sqlite... Oracle is not required

Reply:
@ggovi 

> Provide read-access to the cms-bot means essentially store the password in the repository... Not viable.

I am wondering how, for example, the cms-bot manages to authenticate for using DAS. Are the credential publicly exposed ? @smuzaffar @davidlange6

Reply:
> The O2O tests could be done with frontier+sqlite... Oracle is not required

well, for the DCS yes, for the DAQ O2O no. And also for the DCS we would be testing only half of it, right? The issue with the OCCI library would still go unnoticed.

Reply:
@mmusich , cmsbuild has grid certificate which we use for cmsweb authentication

Reply:
@smuzaffar yes thanks I imagined so, but there's not password associated with it?

Reply:
@mmusich , yes,  no password is associated with it.

Reply:
@smuzaffar OK, thank you.

Reply:
Maybe I should rephrase: "The O2O tests" like all the tests requiring access to the database "MUST be implemented with frontier/sqlite". There is a deployment/distribution issue for the password. No way to store it in github. 
Also, the OCCI problem is in... OCCI. It's an external, I don't think we should have the ambition to test our externals. But we should have the ambition to choose them according to some criteria of maintenance and solidity. From this point of view, we have been discouraging the usage of OCCI since years.